Add paragraph on suggested mapping (#428)
akeneo/pim-helpcenter: content/md/franklin-insights/franklin-insights-mappings.md

### Exact match

> No need to map attributes if they're the same, we do it for you: the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!

Reviewer: ```suggestion
No need to map attributes if they are the same, we do it for you: the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
```
I'd rather use the full word instead of a contraction

![Franklin's exact match screen](../img/franklin_exact_match.png)

::: info

> We consider attributes match if the Franklin attribute code equals the PIM attribute code or PIM attribute label for an English catalog locale.

Reviewer: suggestion ;)
```suggestion
You will have an attribute match if the Franklin attribute code equals the PIM attribute code or its English locale label.
```

:::

### Suggested mapping

> No need to search through a long list of your catalog attributes: if they're close, we suggest the best match for the attribute. The best match is selected, and you can check the other close ones Franklin found. When you save the attribute mapping screen, the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!

Reviewer: Same here 'are' instead of its contraction.
Also suggest to use 'similar' and 'almost identical' instead of 'close'.
Didn't get the sentence "The best match is selected, and you can check the other similar ones Franklin found".

```suggestion
No need to search through a long list of your catalog attributes: if they are almost identical, we suggest the best match for the attribute. The best match is selected, and you can check the other similar ones Franklin found. When you save the attribute mapping screen, the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
```

![Franklin's suggested mapping screen](../img/franklin_suggested_mapping.png)

### Suggest attribute creation

> In case we have no mapping suggestion, we added a new `Create attribute` button. In a single action, you can create the attribute and attach it to the family you're working on without leaving the attribute mapping screen.

Reviewer: ```suggestion
In case we have no mapping suggestion, we added a new `Create attribute` button. In a single action, you can create the attribute and attach it to the family you are working on without leaving the attribute mapping screen.
```

> No need to map attributes if they're the same, we do it for you: the PIM attribute is automatically mapped and saved, the proposal is automatically generated!

Reviewer 2: ```suggestion
No need to map attributes if they're the same, we do it for you: the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
```

![Franklin's exact match screen](../img/franklin_exact_match.png)

::: info

> We consider attributes match if Franklin attribute code equals the PIM attribute code or PIM attribute label for an English catalog locale.

Reviewer 2: ```suggestion
We consider attributes match if the Franklin attribute code equals the PIM attribute code or PIM attribute label for an English catalog locale.
```

:::

### Suggested mapping

> No need to search in the long list of your catalog attributes: if they're close, we suggest the best match for the attribute. The best match is selected, you can check the other close ones we found. When you save the attribute mapping screen, the PIM attribute is automatically mapped and saved, the proposal is automatically generated!

Reviewer 2: ```suggestion
No need to search through a long list of your catalog attributes: if they're close, we suggest the best match for the attribute. The best match is selected, and you can check the other close ones Franklin found. When you save the attribute mapping screen, the PIM attribute is automatically mapped and saved, and a proposal is automatically generated!
```
